cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
```

Comment:
Do we need this file, or can it be removed?
Perhaps just remove comments if it will be needed later?

Reply:
True! I thought we could have them as a form of "documentation" for later. But I suppose I have google:)

Reply:
Yep. You can decide if you want to leave it or not! Doesn't make a huge difference, but it's nice to take a stance whether you want to include vendor comments or not.

Reply:
We have to have the file btw, it comes automatically when running a test.

Reply:
But I will remove the comments